master/buildbot/scripts/populatedatabase.py

```diff
@@ -0,0 +1,206 @@
+# -*- coding: utf-8 -*-
+# This file is part of Buildbot.  Buildbot is free software: you can
+# redistribute it and/or modify it under the terms of the GNU General Public
+# License as published by the Free Software Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright Buildbot Team Members
+from __future__ import print_function
+
+import random
+import sys
+import traceback
+from time import time
+
+import datetime
+import names
+from twisted.internet import defer
+
+from buildbot import config as config_module
+from buildbot.db import connector
+from buildbot.master import BuildMaster
+from buildbot.util import in_reactor
+from buildbot.util import datetime2epoch
+from buildbot.status import results
+
+MAX_UNIQUE_USER_COUNT = 5494
```

Comment:
Magic number?

Reply:
Hm maybe you have right. I've got this number because I've added rows from https://github.com/treyhunner/names/blob/master/names/dist.female.first and https://github.com/treyhunner/names/blob/master/names/dist.male.first. But honestly, I have no idea what can be easily done in code to replace it. Maybe it will be enough to add a comment why exactly this number?

Reply:
Why don't you just put those two files in a set, and `pop` to get an unique name?

Would save you the trouble with the `while` unique checking thing...